.coderabbit.yaml

```
@@ -5,3 +5,5 @@ reviews:
   review_status: true
   auto_review:
     enabled: true
+    branches:
+      - develop
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)